src/Microsoft.SqlTools.Credentials/Microsoft.SqlTools.Credentials.csproj

```diff
@@ -19,7 +19,7 @@
   </ItemGroup>
   <ItemGroup>
 	<PackageReference Include="System.Data.SqlClient" Version="4.4.0-preview1-25305-02" />
-	<PackageReference Include="Microsoft.SqlServer.Smo" Version="140.2.1" />
+	<PackageReference Include="Microsoft.SqlServer.Smo" Version="140.2.2" />
```

Comment:
>140.2.2" [](start = 62, length = 8)

I see an "add" for the new nuget package but I don't see a delete for the older version...could you please delete the older nuget package?